src/skins/tu.tsx

```diff
@@ -260,7 +260,7 @@ export const getTuSkin: GetKnownSkin = () => {
             divider: applyAlpha(palette.white, 0.05),
             dividerInverse: applyAlpha(palette.white, 0.05),
             navigationBarDivider: palette.darkModeBlack,
-            badge: palette.red70,
+            badge: palette.blue30,
```

Comment:
is this change intended? I think the is't enough color contrast to read the number inside the badge now

Reply:
It is, badges are blue in TU brand by design. They have a 6+:1 contrast, so it souldn't be a problem.

Reply:
This is dark mode 🤔 I think Abel is right
https://camo.githubusercontent.com/daa3628439047ee6f4e6ede87c2fb3a069051d80a2aca7b8c83f73619275c1c4/68747470733a2f2f6465766d6973746963617765622e626c6f622e636f72652e77696e646f77732e6e65742f6d6973746963612d7765622d313731393939343934393638382f707269766174652d736b696e2d636f6d706f6e656e74732d73637265656e73686f742d746573742d7473782d636f6d706f6e656e74732d696e2d74752d696e76657273652d616e642d6461726b2d6d6f64652d312d646966662e706e67